MapboxCoreNavigation/NavigationLocationManager.swift

```diff
@@ -10,6 +14,11 @@ open class NavigationLocationManager: CLLocationManager {
     
     var lastKnownLocation: CLLocation?
     
+    /**
+     Indicates whether the device is plugged in or not.
+     */
+    public var isPluggedIn: Bool = false
```

Comment:
Should this public and mutable? If it needs to be public can we make this a getter somehow?